.github/workflows/desktop-file-check.yaml

```diff
@@ -0,0 +1,26 @@
+name: Validate .desktop file
+on:
+  workflow_dispatch: # Manual trigger
+  pull_request:
+    paths:
+      - qt/res/OrganicMaps.desktop
+      - .github/workflows/desktop-file-check.yaml  # Run check on self change
+
+jobs:
+  validate-desktop-file:
+    name: Validate .desktop file
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
```

Comment:
Checking out everything looks like an overkill. Can a single file be checked out? )

Reply:
Yes. Done.